CMakeLists.txt

```diff
@@ -1386,21 +1397,45 @@ if(BUILD_TESTS)
       PROPERTIES FAIL_REGULAR_EXPRESSION "FAILED" TIMEOUT 1000)
   endfunction(configure_test)
 
+  # Installed as part of the installable testsuite.
+  install(TARGETS test_lib
+          LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}/rr)
+  install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/bin/test-monitor
+                   ${CMAKE_CURRENT_BINARY_DIR}/bin/cpuid
+          DESTINATION ${CMAKE_INSTALL_LIBDIR}/rr/testsuite/obj/bin)
+  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/CTestTestfile.cmake
+          DESTINATION ${CMAKE_INSTALL_LIBDIR}/rr/testsuite/obj/)
```

Comment:
I guess you don't want the trailing / here.

Reply:
That's right, I have removed it.